tests/generation/UtilitiesTest.py

```diff
@@ -8,7 +8,9 @@
 from coala_quickstart.generation.Utilities import (
     get_default_args, get_all_args,
     search_for_orig, concatenate, peek,
-    get_language_from_hashbang)
+    get_language_from_hashbang, contained_in)
```

Comment:
dont modify this line; contained_in should be at the beginning of this tuple based on alpha order